### PR TITLE
Added small caption and the H6 Header B to typography.scss and updated the docs to reflect the change

### LINKED ISSUE #759: Add two new text styles
## Description
We need to add two new text styles that are currently available in the [Design Library](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=445%3A1053&viewport=90%2C60%2C0.23130376636981964). These are:
1. Heading 6B
2. Small caption

These styles were created for the mobile resolution where some text styles needed new versions in this resolution. The specs can be found in the mockups.

### PR DESCRIPTION
**Pull request by akshayprabhu200**
## Fixes

Fixes #759 by @panchovm 

## Description

Added both the new fonts to the file and updated the docs.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Please don't commit changes to `package.json` and `package-lock.json` in PRs.

**Comment by akshayprabhu200** [comment]
@dhruvkb I have deleted the file from the commit in the new commit. Is that correct?

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Don't delete `package-lock.json` from the repo, just don't commit it to your PR. See instructions on [this StackOverflow answer](https://stackoverflow.com/a/44355701/2601645). Also there are 3 `package-lock.json` files in the PR, remove all of them.

**Comment by akshayprabhu200** [comment]
Updated the latest PR to match the required changes. Only two changes made, any idea why the deploy is failing though?

File: packages/vocabulary/src/stories/typography.stories.mdx [hunk 1]
```diff
@@ -25,54 +25,48 @@ export const title = () => `
 <p class="title is-3">Title 3</p>
 <p class="title is-4">Title 4</p>
 <p class="title is-5">Title 5</p>
-`
+`;
```
Comment (obulat) [inline packages/vocabulary/src/stories/typography.stories.mdx]: h6 title example should be added here.

File: packages/vocabulary/src/stories/typography.stories.mdx [hunk 2]
```diff
@@ -25,54 +25,48 @@ export const title = () => `
...
 
 export const subtitle = () => `
 <p class="subtitle is-1">Subtitle 1</p>
 <p class="subtitle is-2">Subtitle 2</p>
 <p class="subtitle is-3">Subtitle 3</p>
 <p class="subtitle is-4">Subtitle 4</p>
 <p class="subtitle is-5">Subtitle 5</p>
```
Comment (obulat) [inline packages/vocabulary/src/stories/typography.stories.mdx:36]: h6 subtitle should also be added here.

**Review by obulat — CHANGES_REQUESTED** [review]
There are many formatting changes (extra semicolon, removed newlines, etc. ), @akshayprabhu200 . I guess you ran a linter on the files, could you revert those changes, too, and leave only the changes you've made? It is really difficult to see what you've actually changed this way.
Also, could you add the h6 header style to the storybook? I've added comments inline.

**Comment by akshayprabhu200** [comment]
@obulat added the changes you requested in the stories and reversed Prettier's formatting, it was enabled by default in my VScode config.

**Comment by akshayprabhu200** [comment]
Hi, any update on this?

File: packages/vocabulary/src/stories/typography.stories.mdx [hunk 3]
```diff
@@ -25,6 +25,7 @@ export const title = () => `
 <p class="title is-3">Title 3</p>
 <p class="title is-4">Title 4</p>
 <p class="title is-5">Title 5</p>
+<p class="title is-5">Title 6</p>
```
Comment (obulat) [inline packages/vocabulary/src/stories/typography.stories.mdx]: Classname for Title 6 should probably be `is-6`, not `is-5`.

**Review by obulat — CHANGES_REQUESTED** [review]

**Comment by obulat** [comment]
Hi, @akshayprabhu200 , sorry for the late review, today is the first day after the winter shutdown for CC.
I've left a couple of notes inline. Also, please, revert the changes to the `package.json` and `package-lock.json` files.

**Comment by akshayprabhu200** [comment]
@obulat made the changes you requested in the title (and found the subtitle had the same issue, fixed that too). As for the `package.json` and it's been added by [this commit ](https://github.com/creativecommons/vocabulary/pull/827/commits/3603a3a6e5e131cb51f92b19db5b23b1d22aeeee) which was added to fix update dependencies. Should I reverse that?

**Comment by obulat** [comment]
> @obulat made the changes you requested in the title (and found the subtitle had the same issue, fixed that too). As for the `package.json` and it's been added by [this commit ](https://github.com/creativecommons/vocabulary/pull/827/commits/3603a3a6e5e131cb51f92b19db5b23b1d22aeeee) which was added to fix update dependencies. Should I reverse that?

Oh, sorry about that 🤦 I completely forgot about it: I was reviewing your PR and then forgot to change the branch before pushing this change. Yes, please, could you reverse that commit?

**Comment by akshayprabhu200** [comment]
@obulat Done 👍

**Review by obulat — APPROVED** [review]

**Pull request closed without merging**